multinode-demo/remote_leader.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
```

Comment:
can we add a `[[ -n $FORCE ]] || exit` to this script, and the remote_validator.sh script.  This is not something we want a user to run by accident on their dev machine

Reply:
Yes. I was wondering how to prevent it. This will ensure it.